buildflags/BUILD.gn

```diff
@@ -15,6 +15,7 @@ buildflag_header("buildflags") {
     "ENABLE_VIEW_API=$enable_view_api",
     "ENABLE_PEPPER_FLASH=$enable_pepper_flash",
     "ENABLE_PDF_VIEWER=$enable_pdf_viewer",
+    "ENABLE_PRINTING_ELECTRON=$enable_printing",
```

Comment:
any chance we could merge with the `enable_basic_printing` flag defined in `printing/buildflags/buildflags.gni`?

Reply:
Can we control that flag from our GN files? It's defined [somewhere ](https://cs.chromium.org/chromium/src/printing/buildflags/buildflags.gni?type=cs&q=%22enable_basic_printing+%3D%22&g=0&l=10)as `enable_basic_printing = !is_chromecast && !is_ios && !is_fuchsia`, so I wasn't sure if we could override that or not.

Reply:
yes, anything in a `declare_args()` block will be overridden by `args.gn`.